src/commands/init/index.ts

```diff
@@ -127,8 +217,9 @@ export default class Init extends Command {
         }
     }
 
-    async runPostInitHooks(projectConfig: ProjectConfig, appManifest: any, verbose: boolean) {
-        for (const componentContext of projectConfig.getComponents()) {
+    private async _runPostInitHooks(projectConfig: ProjectConfig, appManifest: any, verbose: boolean) {
```

Comment:
this function needs to be adapted to runPostInitHooks for given ComponentContexts
for singlePackage --> projectConfig.getComponentsForPackageConfig(requestedPackageConfig)
for all --> projectConfig.getComponents()

_runPostInitHooks(componentContexts: ComponentContext[], projectConfig: ProjectConfig, appManifest: any, verbose: boolean)